chain/client/src/client_actor.rs

```diff
@@ -1512,6 +1537,10 @@ impl ClientActor {
             },
         );
     }
+
+    fn handle_catchup_error(&self, err: Error) {
+        error!(target: "client", "{:?} Error occurred during catchup for the next epoch: {:?}", self.client.validator_signer.as_ref().map(|vs| vs.validator_id()), err);
+    }
```

Comment:
is this intended? If so, the name of the function is not optimal -- there is no handling here since it simply logs the error